core/services/workflows/test/wasm/v2/cmd/main.go

```diff
@@ -0,0 +1,45 @@
+//go:build wasip1
```

Comment:
If we can find a way to use the workflow from the other repository, that would be more ideal. I won't block the PR though.